.storybook/main.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ['../ui/app/components/**/*.stories.js'],
+  stories: ['../ui/app/**/**/*.stories.js'],
```

Comment:
Nit: This additional `**` is redundant; `**` is already recursive.

```suggestion
  stories: ['../ui/app/**/*.stories.js'],
```